src/features/internal/cheatcodes/pages/CheatCodes/CheatCodes.native.test.tsx

```diff
@@ -25,26 +25,29 @@ const navigation = {
 } as any // eslint-disable-line @typescript-eslint/no-explicit-any
 
 describe('CheatCodes component', () => {
+  it('should display code push button for testing environment', async () => {
+    env.ENV = 'testing'
+    renderCheatCodes()
+
+    await screen.findByText('CheatCodes')
+
+    expect(screen.queryByText('Check update')).toBeOnTheScreen()
+  })
+
   it.each`
-    environment     | buttonIsdisplayed
-    ${'testing'}    | ${true}
-    ${'staging'}    | ${false}
-    ${'production'} | ${false}
-  `('should display/not display code push button', async ({ environment, buttonIsdisplayed }) => {
+    environment
+    ${'staging'}
+    ${'production'}
```

Comment:
est-ce qu'il serait possible de simplifier en ?

```suggestion
    staging
    production
```

Reply:
TypeScript casse si je l'enlève 🤷

Reply:
🤦